test/integration/task/analytics.js

```diff
@@ -77,5 +77,14 @@ describe('task: analytics', () => {
           // eslint-disable-next-line object-curly-spacing
           au.details.error.should.eql({ testError: 'foo'});
         }))));
+
+
+  it('should check xml content of what analytics reporter sent', testTask(async ({ Configs, analyticsReporter }) => {
+    await Configs.set('analytics', { enabled: true, email: 'test@getodk.org' });
+    await runAnalytics();
+    analyticsReporter.dataSent.startsWith('<?xml version="1.0"?>').should.be.true();
+    analyticsReporter.dataSent.includes('id="odk-analytics"').should.be.true();
```

Comment:
Is it worth checking `version="test-version"` as well? Or is there already test coverage of the form version string?